samples/cpp/facedetect.cpp

```diff
@@ -7,19 +7,21 @@
 using namespace std;
 using namespace cv;
 
-static void help()
+static void help(const char** argv)
 {
     cout << "\nThis program demonstrates the use of cv::CascadeClassifier class to detect objects (Face + eyes). You can use Haar or LBP features.\n"
             "This classifier can recognize many kinds of rigid objects, once the appropriate classifier is trained.\n"
             "It's most known use is for faces.\n"
             "Usage:\n"
-            "./facedetect [--cascade=<cascade_path> this is the primary trained classifier such as frontal face]\n"
-               "   [--nested-cascade[=nested_cascade_path this an optional secondary classifier such as eyes]]\n"
-               "   [--scale=<image scale greater or equal to 1, try 1.3 for example>]\n"
-               "   [--try-flip]\n"
-               "   [filename|camera_index]\n\n"
-            "see facedetect.cmd for one call:\n"
-            "./facedetect --cascade=\"data/haarcascades/haarcascade_frontalface_alt.xml\" --nested-cascade=\"data/haarcascades/haarcascade_eye_tree_eyeglasses.xml\" --scale=1.3\n\n"
+        <<  argv[0]
+        <<  "   [--cascade=<cascade_path> this is the primary trained classifier such as frontal face]\n"
+            "   [--nested-cascade[=nested_cascade_path this an optional secondary classifier such as eyes]]\n"
+            "   [--scale=<image scale greater or equal to 1, try 1.3 for example>]\n"
+            "   [--try-flip]\n"
+            "   [filename|camera_index]\n\n"
+            "see " << argv[0] << ".cmd for one call:\n"
```

Comment:
There is no such `.cmd` file in the repository, please remove it from help message.